docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -16,33 +16,13 @@ Module Support
 Speed Modules
 **************
 
-.. table:: Speed Module Support for DroneCAN
-
-	+--------------+------------------------------------+
-	| Module       | DroneCAN Support                   |
-	+--------------+------------------------------------+
-	| Vertiq 81-08 | .. centered:: |:white_check_mark:| |
-	+--------------+------------------------------------+
-	| Vertiq 40-06 | .. centered:: |:white_check_mark:| |
-	+--------------+------------------------------------+
-	| Vertiq 23-06 | .. centered:: |:x:|                |
-	+--------------+------------------------------------+
+.. include:: ../manual/advanced_only_table.rst
```

Comment:
another common table. This time, we check off all of the advanced speed modules.